### PR TITLE
Relative Abundances Class

### PR DESCRIPTION
**Pull request by profxj**
Class for handling relative abundances.  
Mainly for an AbsSystem.  And maybe 
mainly for DLAs

Tests and doc strings too

Will add a short Notebook and 
an .rst file before merging.

### CODE REVIEW

File: docs/relabund.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Then load data into the data dict.  Here is an example::
+
+   XY._data = {6: dict(flag=1, XH=-1., sigXH=0.2, sig=0.05),
+            8: dict(flag=2, XH=-1.4, sigXH=0.25, sig=0.05),
+            14: dict(flag=1, XH=-1.1, sigXH=0.25, sig=0.05),
+            26: dict(flag=1, XH=-1.4, sigXH=0.25, sig=0.05),
+            32: dict(flag=3, XH=-0.8, sigXH=0.25, sig=0.05),
+            }
+
```
Comment (nhmc) [inline docs/relabund.rst]: What do the flag values mean? They should be described here. Ideally the flag values could be short strings that suggest what they are. e.g. 'lo', 'hi' (or whatever) for lower and upper limits. (But I realise this means making changes elsewhere)

File: docs/relabund.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+More frequent usage will be to instantate using an input
+table of column density measurements, e.g.::
+
+   dla.XY = RelAbund.from_ionclm_table((1,dla.NHI, dla.sig_NHI[0]), dla._ionN)
+
+See pyigm DLA abund Notebook for more.
```
Comment (nhmc) [inline docs/relabund.rst]: Can you link to this here? I couldn't find it with a quick search.

**Comment by nhmc** [comment]
Thanks for the PR. 

Can you give an example of how you would use this to do a quick calculation of an abundance given two column densities? In barak, for example, you do this via:

``` barak.abundances.calc_abund('C', 'Si', 13.5, 13.2)```

File: docs/relabund.rst (hunk 1 above)
Comment (profxj) [inline docs/relabund.rst]: Have added a Table in the doc.

Also, the table() method adds a dict to its meta 
that describes the flagging.

File: docs/relabund.rst (hunk 2 above)
Comment (profxj) [inline docs/relabund.rst]: It is still in a branch..  
I'll try to remember to do this when I merge it.

**Comment by profxj** [comment]
Have just added a new instantiation method which also prints the value
of the pair inputted:

CSi = RelAbund.from_clm_pair('C', 13.5, 'Si', 13.2)

See the updated Notebook

**Comment by nhmc** [comment]
Looks good, thanks for the changes. Please merge when you're ready.

**Comment by profxj** [comment]
merging.

**Pull request merged**